diffkemp/semdiff/result.py

```diff
@@ -44,6 +45,94 @@ class Entity:
         or a parameter.
         If it is a function, it contains the file of the function.
         """
+        class Callstack:
```

Comment:
Don't nest classes this much. Move `Callstack` at least one level up (i.e. directly inside `Result`).

Reply:
Thanks, I moved it to the `Result` class.